### PR TITLE
Standardize pipeline names

### PR DESCRIPTION
**Pull request by scoheb**
Pipeline `metadata.name` should be the same as the `folder` name

### CODE REVIEW

File: catalog/pipeline/fbc-release/0.3/samples/sample_release_PipelineRun.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      value: ""
+    - name: binaryImage
+      value: ""
+    - name: buildTags
+      value: ""
+    - name: buildArches
+      value: ""
+  pipelineRef:
+    name: fbc-release
+    bundle: quay.io/hacbs-release/pipeline-fbc-release:0.2
```
Comment (johnbieren) [inline catalog/pipeline/fbc-release/0.3/samples/sample_release_PipelineRun.yaml:26]: wrong tag

File: catalog/pipeline/fbc-release/0.3/tests/run.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      value: ""
+    - name: binaryImage
+      value: ""
+    - name: buildTags
+      value: ""
+    - name: buildArches
+      value: ""
+  pipelineRef:
+    name: fbc-release
+    bundle: quay.io/hacbs-release/pipeline-fbc-release:0.2
```
Comment (johnbieren) [inline catalog/pipeline/fbc-release/0.3/tests/run.yaml:26]: wrong tag

File: catalog/pipeline/release/0.8/README.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
+# Release Pipeline
+
+Tekton pipeline to release HACBS Snapshot to Quay.
```
Comment (johnbieren) [inline catalog/pipeline/release/0.8/README.md:3]: Maybe this is a good time to change HACBS to Stonesoup?

**Review by johnbieren — CHANGES_REQUESTED** [review]
Wrong tag for the new fbc-release pipeline. I also wonder about the `release` pipeline... It isn't in scope for this PR, but maybe we should have a better name for that one now that we have multiple release pipelines

File: catalog/pipeline/fbc-release/0.3/samples/sample_release_PipelineRun.yaml (hunk 1 above)
Comment (scoheb) [reply catalog/pipeline/fbc-release/0.3/samples/sample_release_PipelineRun.yaml:26]: fixed

File: catalog/pipeline/fbc-release/0.3/tests/run.yaml (hunk 2 above)
Comment (scoheb) [reply catalog/pipeline/fbc-release/0.3/tests/run.yaml:26]: fixed

File: catalog/pipeline/release/0.8/README.md (hunk 3 above)
Comment (scoheb) [reply catalog/pipeline/release/0.8/README.md:3]: fixed

**Comment by scoheb** [comment]
> Wrong tag for the new fbc-release pipeline. I also wonder about the `release` pipeline... It isn't in scope for this PR, but maybe we should have a better name for that one now that we have multiple release pipelines

Yes. I have been bothered by this name for a while. I could change it here in this PR, but that would mean either:
- changing all previous versions
OR
- creating a new folder with version 1

**Review by johnbieren — APPROVED** [review]

File: catalog/pipeline/fbc-release/0.3/fbc-release.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        - name: overwriteFromIndex
+          value: $(params.overwriteFromIndex)
+        - name: binaryImage
+          value: $(params.binaryImage)
+        - name: buildTags
+          value: $(params.buildTags)
+        - name: addArches
+          value: $(params.addArches)
+        - name: requestUpdateTimeout
+          value: $(params.requestUpdateTimeout)
```
Comment (theflockers) [inline catalog/pipeline/fbc-release/0.3/fbc-release.yaml:85]: I think you could already add line you suggested.
```
runAfter:
  - verify-enterprise-contract
```

**Review by theflockers — CHANGES_REQUESTED** [review]

File: catalog/pipeline/fbc-release/0.3/fbc-release.yaml (hunk 4 above)
Comment (scoheb) [reply catalog/pipeline/fbc-release/0.3/fbc-release.yaml:85]: fixed

**Comment by scoheb** [comment]
> > Wrong tag for the new fbc-release pipeline. I also wonder about the `release` pipeline... It isn't in scope for this PR, but maybe we should have a better name for that one now that we have multiple release pipelines
> 
> Yes. I have been bothered by this name for a while. I could change it here in this PR, but that would mean either:
> 
> * changing all previous versions
>   OR
> * creating a new folder with version 1

I will address this in a future PR.

**Pull request merged**